src/lavinmq/queue/message_store.cr

```diff
@@ -317,7 +317,22 @@ module LavinMQ
             file.write_bytes Schema::VERSION
             @replicator.try &.append path, Schema::VERSION
           else
-            SchemaVersion.verify(file, :message)
+            begin
+              SchemaVersion.verify(file, :message)
+            rescue EmptyFile
+              # delete empty file, it will be recreated if it's needed
+              Log.warn { "Empty file at #{path}, deleting it" }
+              file.delete.close
+              @replicator.try &.delete_file(path)
+              if idx == 0
```

Comment:
how come we recreate the first file but not others? Maybe add a comment about that.

Reply:
We need at least one segment to exist, otherwise we get errors in a lot of places (for example where we use `first_/last_key` & `first_/last_value`). And for a file that isn't the first I don't think we need to recreate it? Better to let it be recreated when/if needed IMO. 

Maybe could be changed to check if it's the only segment instead? But I don't see how we would end up with an empty file followed by other files. 

Added a comment about it!